test/parallel/test-https-close.js

```diff
@@ -37,7 +37,7 @@ function shutdown() {
   }
 }
 
-server.listen(0, function() {
+server.listen(0, () => {
   const requestOptions = {
     hostname: '127.0.0.1',
     port: this.address().port,
```

Comment:
This function uses `this`, which is affected by the `function` vs arrow function distinction – I think you need to change this refernce?

```suggestion
    port: server.address().port,
```

Reply:
Done. Thanks.